Cargo.toml

```diff
@@ -21,11 +21,13 @@ name = "string_cache"
 [features]
 serde_support = ["serde"]
 default = ["serde_support"]
+malloc_size_of = ["dep:malloc_size_of", "malloc_size_of_derive"]
```

Comment:
Use of `dep:` syntax requires Rust `1.71`, so we should bump the `rust-version` if we're going to do this. I think this should be fine for `string_cache`? But perhaps worth noting I had to revert to the older feature syntax for crates like `euclid` that have lower MSRVs.